types.proto

```diff
@@ -34,8 +34,8 @@ message MarketWithFee {
 }
 
 message Price {
-  float base_price = 1;
-  float quote_price = 2;
+  double base_price = 1;
+  double quote_price = 2;
```

Comment:
I would rather do 
```suggestion
  float base_price_deprecated = 1;
  float quote_price_deprecated = 2;
  double base_price = 3;
  double quote_price = 4;
```

AFAIK protobuf binary encoding only cares about the number and will keep working, even if change names

Reply:
https://stackoverflow.com/a/45431953

Reply:
Done, @tiero please review.